crs_motion_planning/src/motion_planning_server.cpp

```diff
@@ -514,28 +536,28 @@ class MotionPlanningServer : public rclcpp::Node
                   std::shared_ptr<crs_msgs::srv::RobotPositioner::Response> response)
   {
     Eigen::Isometry3d ur_base_pose = Eigen::Isometry3d::Identity();
-    ur_base_pose.rotate(Eigen::Quaterniond(0.7071, 0, 0.7071, 0));
+    ur_base_pose.rotate(Eigen::Quaterniond(0.5, -0.5, -0.5, -0.5));
```

Comment:
I know at the moment is ur specific but to the extend that we can we should make it robot agnostic.  Will this rotation value apply only to the WSU workcell?

Reply:
This is the rotation between the robot stand and the base, so not UR specific, but yes this is WSU workcell specific based on the rotation.

Reply:
Not sure that this is a good approach as it locks us into just one workcell

Reply:
Should we add a configuration file that has all the base locations in it?

Reply:
Personally I think if it's decided to use a new workcell in the future and that becomes variable then we add a configuration for it, otherwise I think it is okay to leave hardcoded. Seems like it'll be a lot of effort some something that isn't already considered a necessary change.

Reply:
As discussed offline, having hardcoded position values that are specific to one workcell limits portability to other workcells, or even changes in the urdf could render this unusable.  This change may also prevent running in simulation which is useful to have in order to test various things.
One way to handle this in a more general way is to setup a prismatic joint between the rail and the robot base and tie integer rail positions to specific joint values along the rail which could be read from a yaml file. 
I'll merge for now due to the urgency of this but we should create an issue.